card.css

```diff
@@ -1 +1,186 @@
 @import "base.css";
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap");
+
+.card {
+  display: grid;
+  place-items: end center;
+  row-gap: 40px;
+}
+.card-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 70px;
+  font-family: "Montserrat", sans-serif;
+}
+.card-design {
+  width: 500px;
+  height: 300px;
+  border-radius: 24px;
+  box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+}
+.card-design--one {
+  background: url(./assets/images/bg-design-1.png);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.card-design--two {
+  background: url(./assets/images/bg-design-4.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.card-design--three {
+  background: conic-gradient(
+    from 228.45deg at 61.72% 26.04%,
+    #552e8c -20.95deg,
+    #884fd7 30.59deg,
+    #7b40ce 105.58deg,
+    #c0c0c0 156.31deg,
+    #5d2da2 193.74deg,
+    #552e8c 339.05deg,
+    #884fd7 390.59deg
+  );
+}
+.card-design--four {
+  background: url(./assets/images/bg-design-2.png);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.card-design--five {
+  background: url(./assets/images/bg-design-3.png);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.card-design--six {
+  background: radial-gradient(
+    50.05% 57.8% at 50.05% 42.2%,
+    #ff4aac 3.03%,
+    #a31863 100%
+  );
+}
+
+.card-design-expires {
+  float: right;
+  color: var(--white);
+  line-height: 5px;
+  margin-top: 32px;
+  margin-right: 20px;
+}
+.card-design-expires p {
+  font-size: 12px;
```

Comment:
Also moving forward, use rem or % for units
Instead of 5px, make it 5rem instead.
You can read more on why this is better than px units - [link](https://chiamakaikeanyi.dev/sizing-in-css-px-vs-em-vs-rem/)

Reply:
Okay I'll make changes to it

Reply:
@VictoriaAde i've made changes kindly review. Thanks.